code/datums/supplypacks/contraband_vr.dm

```diff
@@ -4,4 +4,16 @@
 	cost = 150
 	containertype = /obj/structure/closet/crate
 	containername = "Stolen crate"
-	contraband = 1
\ No newline at end of file
+	contraband = 1
+
+/datum/supply_packs/supply/awoo
```

Comment:
These should be named properly. Not mlem or awoo. Catgirl or Wolfgirl. Or awoo I guess because the file path is awoo.